test/specs/global.deprecations.tests.js

```diff
@@ -163,6 +163,43 @@ describe('Deprecations', function() {
 				});
 			});
 		});
+
+		describe('Time Axis: unitStepSize option', function() {
+			function createScale(data, options) {
+				var scaleID = 'myScale';
+				var mockContext = window.createMockContext();
+				var Constructor = Chart.scaleService.getScaleConstructor('time');
+				var scale = new Constructor({
+					ctx: mockContext,
+					options: options,
+					chart: {
+						data: data
+					},
+					id: scaleID
+				});
+
+				scale.update(400, 50);
+				return scale;
+			}
+
+			it('should use the stepSize property', function() {
+				var mockData = {
```

Comment:
Could we simply acquire a chart with minimal options and remove the `createScale` setup?